librecad/src/main/qc_applicationwindow.cpp

```diff
@@ -1484,6 +1488,9 @@ QC_MDIWindow* QC_ApplicationWindow::slotFileNew(RS_Document* doc) {
     connect(view, SIGNAL(gridStatusChanged(const QString&)),
             this, SLOT(updateGridStatus(const QString&)));
 
+    connect(view, SIGNAL(status_bar_widgets_visibility_request(bool)),
+            this, SLOT(status_bar_widgets_visibility(bool)));
```

Comment:
The deprecated macro syntax should not be used any longer.
The preferred syntax is:
```
    connect( view, &QG_GraphicView::status_bar_widgets_visibility_request,
                   this, &QC_ApplicationWindow::status_bar_widgets_visibility);
```
This is type save and detects issues at compile time, not at run time, like the macro syntax.

This applies to new code, existing code with macro syntax should not be touched, until there are strong reasons to update.

Reply:
I did not know that. I basically copy-pasted a similar code a couple lines above, and changed the variables.